markdownlint.js

```diff
@@ -24,6 +24,13 @@ function jsYamlSafeLoad(text) {
   return require('js-yaml').load(text);
 }
 
+const exitCodes = {
+  lintFindings: 1,
```

Comment:
"lintFailures", maybe?

Reply:
"lintFailures" suggests to me that linting failed to happen, aka an unexpected failure, hence "lintFindings"

This one's a tricky terminology issue and is also frustratingly non standard across a lot of linters :(

Reply:
lintIssues?